### PR TITLE
Adding dumb component for simple links and buy button dummy text

### PR DESCRIPTION
**Pull request by marina-marina**
image

- Added a paragraph of dummy text instead of buy button for now
- Added dumb component for simple links

### CODE REVIEW

File: src/components/SimpleLink/SimpleLink.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes as toBe } from 'react';
+import './simple-link.scss';
+import Link from 'gatsby-link';
+
+export default class Header extends React.Component {
```
Comment (viattik) [inline src/components/SimpleLink/SimpleLink.js]: image

Comment (viattik) [reply src/components/SimpleLink/SimpleLink.js]: And basically it's not a simple link, it's kinda `target="_blank"`. What do you think about moving `target` to props?

**Comment by viattik** [comment]
do we have that light blue color in our styles?
I think we should use either green or white.
Original conference's site uses green one: https://yougottalovefrontend.com/schedule

File: src/components/SimpleLink/SimpleLink.js (hunk 1 above)
Comment (marina-marina) [reply src/components/SimpleLink/SimpleLink.js]: good idea

**Pull request merged**